chapter_common_operators/matmul.md

```diff
@@ -26,11 +26,11 @@ def matmul(n, m, l):
     C : n x m matrix with C = A B
     """
     k = tvm.reduce_axis((0, l), name='k')
-    A = tvm.placeholder((n, l), name='A')
-    B = tvm.placeholder((l, m), name='B')
+    A = tvm.placeholder((n, l), name='a')
```

Comment:
Upper cases are for symbolic tensors, lower cases are used by ndarrays.

Reply:
Makes sense, will change them back.